projects/commudle-admin/src/app/components/home/components/home-communities/home-communities.component.scss

```diff
@@ -1,100 +1,103 @@
-@import "projects/commudle-admin/src/assets/styles/variables";
-
-nb-card-header, div, a {
-  font-family: $primary-font-family;
-}
-
-.home-communities {
-  margin: $space-16 0 0;
-  border: 0;
-  border-radius: $border-10;
-
-  nb-card-header {
-    font-size: $fs-26;
-    font-weight: $fw-600;
-    color: $gunmetal;
-    padding: $space-32 $space-28 $space-16;
-    border-bottom: none;
-  }
-
-  .home-communities-list {
-    padding: 0;
-
-    nb-list {
-      nb-list-item {
-        padding: $space-16;
-        margin-bottom: $space-8;
-        display: flex;
-        justify-content: space-between;
-        align-items: center;
-        border: none;
-
-        app-skeleton-text {
-          width: 100%;
-        }
-
-        .home-community {
-          display: flex;
-          align-items: center;
-          margin-right: $space-8;
-
-          .home-community-image {
-            display: flex;
-            align-items: center;
-            min-height: 2.5rem;
-            max-height: 2.5rem;
-            min-width: 2.5rem;
-            max-width: 2.5rem;
-            overflow: hidden;
-            border-radius: 50%;
-            border: 0.5px solid #E9E8E8;
-            margin-right: $space-8;
-
-            img {
-              width: 100%;
-            }
-          }
-
-          .home-community-details {
-            .home-community-name {
-              font-size: $fs-14;
-              font-weight: 600;
-              color: $gunmetal;
-            }
-
-            .home-community-member-count {
-              font-size: $fs-12;
-              font-style: italic;
-              color: $spanish-gray;
-            }
-          }
-        }
-
-        .home-communities-join {
-          padding: $space-4 $space-12;
-          font-family: $primary-font-family;
-          font-weight: 400;
-          text-transform: capitalize;
-          color: white;
-          background-color: $neon-blue;
-          border: none;
-        }
-
-        .home-communities-join.active {
-          color: $neon-blue;
-          background-color: white;
-          border: 0.5px solid $neon-blue;
-        }
-      }
-    }
-
-    .home-communities-all {
-      padding: $space-8 $space-28 $space-24;
-      text-align: center;
-
-      a {
-        color: rgba($neon-blue, 0.95);
-      }
-    }
-  }
-}
+@import "projects/commudle-admin/src/assets/styles/variables";

```

Comment:
Why is this recreated too :P

Reply:
I think that was because of some line ending issue on my side. I've fixed it now, so wouldn't happen again!